test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py

```diff
@@ -25,34 +25,57 @@ class AccountConnectionWizard:
         "visible": 1,
         "window": names.error_QMessageBox,
     }
+    URL_LOCKED = {
+        "name": "urlLabel",
+        "type": "QLabel",
+        "visible": 1,
+        "window": names.owncloudWizard_OCC_OwncloudWizard,
+    }
```

Comment:
`unlocked` and `locked` icon has the same name

Reply:
Since the element is same for both locked and unlocked icon I have used a vp to check for a locked icon.